app/views/homes/show.html.erb

```diff
@@ -1,25 +1,31 @@
-<%= simple_form_for TelevisionShow.new do |form| %>
-  <%= form.input :title %>
-  <%= form.button :submit %>
-<% end %>
+<main>
+  <h2>Recommendations</h2>
+  <%= simple_form_for TelevisionShow.new do |form| %>
+    <%= form.input :title, label: false, placeholder: "The Wire" %>
```

Comment:
I feel like `The Wire` is a serious default. What about something more frivolous?

Also, this should be a constant, right?